registrations/admin.py

```diff
@@ -87,3 +100,73 @@ def get_readonly_fields(self, request, obj=None):
 
 
 admin.site.register(Registration, RegistrationAdmin)
+
+
+class DefaultPriceGroupListFilter(admin.EmptyFieldListFilter):
+    def __init__(self, field, request, params, model, model_admin, field_path):
+        super().__init__(field, request, params, model, model_admin, field_path)
+
+        self.title = _("Is default")
+
+    def choices(self, changelist):
+        for lookup, title in (
+            (None, _("All")),
+            ("1", _("Yes")),
+            ("0", _("No")),
+        ):
+            yield {
+                "selected": self.lookup_val == lookup,
+                "query_string": changelist.get_query_string(
+                    {self.lookup_kwarg: lookup}
+                ),
+                "display": title,
+            }
+
+
+class PriceGroupAdmin(VersionAdmin):
```

Comment:
You could add `TranslationAdmin` class (you can import it from `modeltranslation.admin`) to display language specific fields for the description in Django admin as well. 

After adding `TranslationAdmin` you have to move "modeltranslation" before "helusers.apps.HelusersAdminConfig" in INSTALLED_APPS to avoid `The model "Registration" is not registered for translation.` errors.

`TranslationAdmin` should be added to RegistrationAdmin class as well. But that can be done in a separate PR.

Reply:
A very important point - I've now implemented these changes (also for `RegistrationAdmin`)